.github/labeler.yml

```diff
@@ -100,6 +100,12 @@
   - "**/*bluetooth*"
 "area: Bluetooth Mesh":
   - "subsys/bluetooth/mesh/**/*"
+"area: Bluetooth Controller":
+  - "subsys/bluetooth/controller/**/*"
+"area: Bluetooth Host":
+  - "subsys/bluetooth/host/**/*"
+  - "subsys/bluetooth/mesh/**/*"
```

Comment:
I would suggest to remove mesh from here, since it already has it's own label, (see above).

Reply:
After the "area" prefix has been added, this certainly makes sense. Will update.